app/src/main/java/org/systers/mentorship/view/activities/FeedbackActivity.kt

```diff
@@ -0,0 +1,216 @@
+package org.systers.mentorship.view.activities
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.text.Editable
+import org.systers.mentorship.R
+import android.util.Patterns
+import androidx.core.app.ComponentActivity
+import androidx.core.app.ComponentActivity.ExtraData
+import androidx.core.content.ContextCompat.getSystemService
+import android.text.TextUtils
+import android.text.TextWatcher
+import android.util.Log
+import android.view.View
+import android.widget.ImageButton
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_feedback.*
+
+/*
+* Author: Abhinav Srivastava
+* Display Name: asawesome07
+* email ID: abhinav.wii@gmail.com*/
```

Comment:
no need to put this here, it should be removed, the credits will be associated on the commit